app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -261,7 +261,7 @@ fun HomeScreen(
         sheetBackgroundColor = Color.White,
         sheetState = addSheetState,
         sheetContent = {
-            AddFavoritesSearchSheet(routeViewModel = routeViewModel) {
+            AddFavoritesSearchSheet(homeViewModel=homeViewModel, favoritesViewModel=favoritesViewModel) {
 
```

Comment:
The spacing here is strange; please format! `Ctrl + Alt + L`

Also, this composable is getting pretty cluttered with sheet and screen logic. We have a `BottomSheetScaffold` AND a `ModalBottomSheetLayout`? Not your fault or anything, just complaining out loud, lol. Let me make an issue for this.

Reply:
Issue created!